src/Compatibility/Core/src/AppHostBuilderExtensions.cs

```diff
@@ -57,6 +57,9 @@ internal static void ResetCompatibilityCheck()
 			Controls.Hosting.CompatibilityCheck.ResetCompatibilityCheck();
 		}
 
+		[Obsolete("The compatibility functionality is deprecated. This will be removed in the future, please make sure to migrate to handlers. " +
+			"You might already not be using any of the compatibility functionality anymore, in that case please remove this call " +
+			"(and the reference to the Compatibility NuGet package) as it will introduce unnecessary overhead that impacts performance.")]
 		public static MauiAppBuilder UseMauiCompatibility(this MauiAppBuilder builder)
```

Comment:
How about including a link to the migration from Renderers to Handlers?
```
[Obsolete("The compatibility functionality is deprecated and will be removed in future versions. Please migrate to handlers for improved support and performance. " +
          "Refer to the migration guide at https://learn.microsoft.com/en-us/dotnet/maui/migration/renderer-to-handler." +
          "If compatibility functionality is no longer in use, remove this call and the reference to the Compatibility NuGet package to avoid unnecessary overhead.")]
```

Reply:
Should these messages be translated ? and live in a resources file?